src/plugins/data/public/ui/saved_query_management/saved_query_management_component.tsx

```diff
@@ -208,150 +191,130 @@ export function SavedQueryManagementComponent({
   };
 
   return (
-    <Fragment>
-      <EuiPopover
```

Comment:
Was this taken out of a popover? If so, why are we still using popover components inside?

Judging from video, is this just replacing content of current context menu?

Reply:
yes, instead of only rendering the filter menu options, this filter popover will now need to render 3 components, filter options, save query panel, and add filter panel. This separation is to make things easier. It's replacing the current context menu, I leave the other popover components unchanged, so that when the toggle is off, the existing experiences remains the same

Reply:
I missed filter_options.tsx before i think which would have helped make sense of this.